bin/sozo/src/commands/build.rs

```diff
@@ -36,6 +41,15 @@ impl BuildArgs {
             builtin_plugins.push(BuiltinPlugins::Unity);
         }
 
+        if self.stats {
+            let target_dir = &compile_info.target_dir;
+            let contracts_statistics: Vec<ContractStatistics> =
+                get_contract_statistics_for_dir(target_dir)
+                    .expect(format!("Error getting contracts in dir {target_dir}").as_str());
```

Comment:
imo sometimes its better to rely on type inference to make the formatting nicer

```suggestion
            let contracts_statistics = get_contract_statistics_for_dir(target_dir)
                .context(format!("Error getting contracts stats"))?
```